src/components/bookItem.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const BookItem = () => (
+  <li>
+    <button type="button">Remove</button>
+  </li>
+);
+
+export default BookItem;
```

Comment:
- [ ] It might be a good idea that `BookItem` component takes the book data as a prop and renders the title, author and its remove button too. 😃